README.md

```diff
@@ -54,7 +54,7 @@ BiocManager::install("miaSim")
 
 miaSim is accompanied by miaSimShiny, which allows users to explore the parameter space of their models in real-time in an intuitive graphical interface. miaSimShiny is available at https://github.com/gaoyu19920914/miaSimShiny.
 
-Follow [this link](https://gaoyu.shinyapps.io/shiny_rep/) to try the online version of miaSimShiny.
+Follow [this link](https://gaoyu.shinyapps.io/shiny_rep/) to try the online version.
```

Comment:
Rephrase this as: 

You can experiment with miaSimShiny in an [interactive online application](https://gaoyu.shinyapps.io/shiny_rep/).